sql-mysql-postgresql-nosql.mdx

```diff
@@ -0,0 +1,224 @@
+---
+title: 'Navigating the database landscape: overview of SQL, MySQL, PostgreSQL, and NoSQL'
+description: 'An exploration of some database options and discussion on where and when to use them.'
```

Comment:
Because the description is displayed as a subtitle, I think it's an opportunity to use more SEO keywords. If you feel like that would be too much, then the current description is nice in balancing it.